docs/getting_started.rst

```diff
@@ -67,18 +157,14 @@ Once that's finished building, you can source the new colcon overlay:
 
     source install/setup.bash
 
-To get the soss-ros1 plugin built, you should create a new workspace and go to it:
+Notice, with reference to table .., that you now have both SOSS and the SOSS-ROS2 SH installed.
+To get the SOSS-ROS1 System Handle built, you need to create a new workspace, and then clone the dedicated repo into it:
 
 .. code-block:: bash
 
     cd ..
-    mkdir soss-ros1
-    cd soss-ros1
-
-Clone the `soss-ros1` plugin which is hosted in a different repo:
-
-.. code-block:: bash
-
+    mkdir workspaces/soss-ros1
+    cd workspaces/soss-ros1
```

Comment:
Following the old steps will result in a workspace with the structure:

    workspaces/soss/src/soss
    workspaces/soss-ros1/src/soss-ros1

Which is nice but following the new...

    workspaces/soss/src/soss
    workspaces/soss/workspaces/soss-ros1/src/soss-ros1

Which is confusing... and ugly :P

Reply:
ok, changed to how it was before. check please! I'm not sure wether I did it right.

Reply:
LG